### PR TITLE
add live update version

### PR DESCRIPTION
**Pull request by jazzdan**
Only thing that changes is the Tiltfile

### CODE REVIEW

File: 04_live_update/Tiltfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+docker_build("frontend", ".", dockerfile="frontend.dockerfile",
+  live_update=[
+    sync('./frontend', '/app/frontend'),
```
Comment (l-korbes) [inline 04_live_update/Tiltfile]: Won't this cause the old binary to be synced *and then* the new binary gets compiled? Or do things not happen in the order they appear here?

**Comment by landism** [comment]
> Only thing that changes is the Tiltfile

maybe it'd be good to do two-commit PRs for work here: one commit that copies the previous step to the new directory, and one commit that changes it

File: 04_live_update/Tiltfile (hunk 1 above)
Comment (landism) [reply 04_live_update/Tiltfile]: the idea with this is that we never build the binary locally - we're only syncing code

Comment (l-korbes) [reply 04_live_update/Tiltfile]: Ohhh, this runs inside the container—got it.

**Pull request merged**